backend/codes/migrations/0006_auto_20210321_1958.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.7 on 2021-03-21 19:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('codes', '0005_auto_20210315_2145'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='code',
+            name='date_approved',
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+    ]
```

Comment:
Czy potrzebna jest ta migracja ? Trochę randomowo tutaj się znalazła

Reply:
A to czasem nie przez to że nie mamy folderu migracji w .gitignore?

Reply:
https://github.com/xszym/UMCS-lights/pull/15#discussion_r594699407

Reply:
Jak coś to ja nie próbuję się kłócić bo nawet nie wiem do końca do czego te migracje są, tak tylko chcę w razie czego zwrócić uwagę xD

Reply:
Migracje są nam po to, że jak zmienia się model w bazie danych, możemy dopisać logikę wypełnienia tych pól jeśli dane już są w bazie.

Pytam się, ponieważ nigdzie w tym PR Szymon nie robił żadnych zmian na modelu a migracja się pojawiła.

Reply:
wywalą ją :D